static/js/controllers/guided-setup-modal.js

```diff
@@ -0,0 +1,151 @@
+var libphonenumber = require('libphonenumber/utils'),
+    countries = require('../modules/countries');
+
+// TODO convert this controller to use angular more and jquery less
```

Comment:
Because guided setup isn't going to be used in the forseeable future the issue would just get closed as won't fix. I left the TODO in here so if anyone else sees it they'll at least know that I know it sucks.

The good news - if it's never used the jquery is never run! Also it works just fine, it's just not following the angular first strategy we've adopted.